src/fights.ts

```diff
@@ -420,11 +420,11 @@ export function dailyFights(): void {
         nextFight = getNextEmbezzlerFight();
 
         // try to deliver the thesis
-        const romanticMonsterPossible =
+        const romanticMonsterImpossible =
           (getCounter("Romantic Monster Window end") === -1 &&
             getCounters("Romantic Monster Window end", -1, -1).trim() === "") ||
```

Comment:
Doesnt `getCounter("Romantic Monster Window end") === -1 && getCounters("Romantic Monster Window end", -1, -1).trim() === "") ` imply we are past the romantic window?

Reply:
That implies that the window end counter does not exist, which means that we're probably done

Reply:
Ah, that tracks, and makes the logic make sense.